# Xenacoelomorph Neuropeptidomes Reveal a Major Expansion of Neuropeptide Systems during Early Bilaterian Evolution - Supplementary Data

This GitHub repository provides the supplementary data and the custom code referenced in the publication cited below.

## How to use and cite these files 

All files are publicly available and can be used for further research or other applications. However, if you utilize these resources in your work, we kindly request that you cite our original publication.

**Xenacoelomorph Neuropeptidomes Reveal a Major Expansion of Neuropeptide Systems during Early Bilaterian Evolution.** Daniel Thiel, Mirita Franz-Wachtel, Felipe Aguilera, Andreas Hejnol. *Molecular Biology and Evolution* **35**:2528-2543 (2018). https://doi.org/10.1093/molbev/msy160

**Abstract**

Neuropeptides are neurosecretory signaling molecules in protostomes and deuterostomes (together Nephrozoa). Little, however, is known about the neuropeptide complement of the sister group of Nephrozoa, the Xenacoelomorpha, which together form the Bilateria. Because members of the xenacoelomorph clades Xenoturbella, Nemertodermatida, and Acoela differ extensively in their central nervous system anatomy, the reconstruction of the xenacoelomorph and bilaterian neuropeptide complements may provide insights into the relationship between nervous system evolution and peptidergic signaling. Here, we analyzed transcriptomes of seven acoels, four nemertodermatids, and two Xenoturbella species using motif searches, similarity searches, mass spectrometry and phylogenetic analyses to characterize neuropeptide precursors and neuropeptide receptors. Our comparison of these repertoires with previously reported nephrozoan and cnidarian sequences shows that the majority of annotated neuropeptide GPCRs in cnidarians are not orthologs of specific bilaterian neuropeptide receptors, which suggests that most of the bilaterian neuropeptide systems evolved after the cnidarianâ€“bilaterian evolutionary split. This expansion of more than 20 peptidergic systems in the stem leading to the Bilateria predates the evolution of complex nephrozoan organs and nervous system architectures. From this ancient set of neuropeptides, acoels show frequent losses that correlate with their divergent central nervous system anatomy. We furthermore detected the emergence of novel neuropeptides in xenacoelomorphs and their expansion along the nemertodermatid and acoel lineages, the two clades that evolved nervous system condensations. Together, our study provides fundamental insights into the early evolution of the bilaterian peptidergic systems, which will guide future functional and comparative studies of bilaterian nervous systems.

## Author contact

- [Daniel Thiel](mailto:daniel.thiel@uib.no) (first author - PhD student)
- [Felipe Aguilera](mailto:felipe.aguilera@uib.no) (senior author - PostDoc)
- [Andreas Hejnol](mailto:andreas.hejnol@uib.no) (corresponding author)

## How to download data files

The files in this repository are ready for use. Simply click on the file, and you will be directed to the GitHub webpage to save it to your device.

## Index of data file contents

[01-Supplementary-Figures.pdf](https://github.com/faguil/Xenacoelomorph-Neuropeptides-Evolution/blob/main/01-Supplementary-Figures.pdf) contains some of the supplementary figures used in this study.

- **Supplementary Figure 1.** Conserved metazoan neuropeptides of xenacoelomorphs.
- **Supplementary Figure 2.** Precursor structure and peptide sequence logo representations of xenacoelomorph-specific multi-copy peptides.
- **Supplementary Figure 3.** Flux map of the central carbon metabolism for *Tetraselmis* sp. M8 under nitrogen limiting conditions at 72 h.
- **Supplementary Figure 4.** Distribution of Gene ontology (GO) terms assigned to annotated differentially expressed reads (DERs) in Control and Nitrogen-starved treatments.
- **Supplementary Figure 5.** Growth and lipid accumulation of *Tetraselmis* sp. M8 in a time-course experiment using control and nitrogen-starved cultures.
- **Supplementary Figure 6.** Flux map of the central carbon metabolism under the lipid accumulation for *Tetraselmis* sp. M8 under nitrogen limiting conditions at 24 h.
- **Supplementary Figure 7.** qRT-PCR data of fatty acid pathway genes.
- **Supplementary Figure 8.** qRT-PCR data of TAG pathway genes.
- **Supplementary Figure 9.** qRT-PCR data of lipid catabolism pathway genes.

## How to download code files

The code in this repository is all stored in the 00-Code folder, and can be downloaded as described above. Most scripts were built in R or Python and are therefore in .R, .py or .ipynb format, which can be visualised in your IDE of choice (e.g., RStudio, Visual Studio Code, etc.). Some of them, specially those that combine multiple steps across different programming languages or softwares called in the command line terminal, were put together in .txt or .docx files, from which the different code chunks can be copied.

We believe that the script file names and the comments in the scripts make them self-explanatory, but please do not hesitate to reach out if you have any doubts or questions.
