# Xenacoelomorph Neuropeptidomes Reveal a Major Expansion of Neuropeptide Systems during Early Bilaterian Evolution - Supplementary Data

This GitHub repository provides the supplementary data and the custom code referenced in the publication cited below.

## How to use and cite these files 

All files are publicly available and can be used for further research or other applications. However, if you utilize these resources in your work, we kindly request that you cite our original publication.

**Xenacoelomorph Neuropeptidomes Reveal a Major Expansion of Neuropeptide Systems during Early Bilaterian Evolution.** Daniel Thiel, Mirita Franz-Wachtel, Felipe Aguilera, Andreas Hejnol. *Molecular Biology and Evolution* **35**:2528-2543 (2018). https://doi.org/10.1093/molbev/msy160

**Abstract**

Neuropeptides are neurosecretory signaling molecules in protostomes and deuterostomes (together Nephrozoa). Little, however, is known about the neuropeptide complement of the sister group of Nephrozoa, the Xenacoelomorpha, which together form the Bilateria. Because members of the xenacoelomorph clades Xenoturbella, Nemertodermatida, and Acoela differ extensively in their central nervous system anatomy, the reconstruction of the xenacoelomorph and bilaterian neuropeptide complements may provide insights into the relationship between nervous system evolution and peptidergic signaling. Here, we analyzed transcriptomes of seven acoels, four nemertodermatids, and two Xenoturbella species using motif searches, similarity searches, mass spectrometry and phylogenetic analyses to characterize neuropeptide precursors and neuropeptide receptors. Our comparison of these repertoires with previously reported nephrozoan and cnidarian sequences shows that the majority of annotated neuropeptide GPCRs in cnidarians are not orthologs of specific bilaterian neuropeptide receptors, which suggests that most of the bilaterian neuropeptide systems evolved after the cnidarianâ€“bilaterian evolutionary split. This expansion of more than 20 peptidergic systems in the stem leading to the Bilateria predates the evolution of complex nephrozoan organs and nervous system architectures. From this ancient set of neuropeptides, acoels show frequent losses that correlate with their divergent central nervous system anatomy. We furthermore detected the emergence of novel neuropeptides in xenacoelomorphs and their expansion along the nemertodermatid and acoel lineages, the two clades that evolved nervous system condensations. Together, our study provides fundamental insights into the early evolution of the bilaterian peptidergic systems, which will guide future functional and comparative studies of bilaterian nervous systems.

## Author contact

- [Daniel Thiel](mailto:daniel.thiel@uib.no) (first author - PhD student)
- [Felipe Aguilera](mailto:felipe.aguilera@uib.no) (senior author - PostDoc)
- [Andreas Hejnol](mailto:andreas.hejnol@uib.no) (corresponding author)

## How to download code files

The code in this repository is stored in the [00-Code folder](https://github.com/faguil/Xenacoelomorph-Neuropeptides-Evolution/tree/main/00-Code) and can be downloaded as outlined below. The ***protein_motif_searching.pl*** script, written in Perl, scans one or more protein motif patterns in FASTA files. For detailed instructions on how to use the script, please refer to the README file in the 00-Code folder. While we believe the script and accompanying README make it largely self-explanatory, do not hesitate to reach out if you have any questions or concerns.

## How to download data files

The files in this repository are ready for use. Simply click on the file, and you will be directed to the GitHub webpage to save it to your device.

## Index of data file contents

[01-Supplementary-Figures.pdf](https://github.com/faguil/Xenacoelomorph-Neuropeptides-Evolution/blob/main/01-Supplementary-Figures.pdf) contains some of the supplementary figures used in this study.

- **Supplementary Figure 1.** Conserved metazoan neuropeptides of xenacoelomorphs.
- **Supplementary Figure 2.** Precursor structure and peptide sequence logo representations of xenacoelomorph-specific multi-copy peptides.
- **Supplementary Figure 3.** Phylogenetic analysis of neuropeptide GPCRs using FastTree.

[02-CLANS-GPCRs.pdf](https://github.com/faguil/Xenacoelomorph-Neuropeptides-Evolution/blob/main/02-CLANS-GPCRs.pdf) contains the cluster analysis of neuropeptide GPCRs.

[03-Xenacoelomorph-neuropeptide-precursors-and-reference-sequences.pdf](https://github.com/faguil/Xenacoelomorph-Neuropeptides-Evolution/blob/main/03-Xenacoelomorph-neuropeptide-precursors-and-reference-sequences.pdf) contains the predicted xenacoelomorph preproneuropeptides and reference sequences used in this study.

[04-Xenacoelomorph_neuropeptide_GPCRs.txt](https://github.com/faguil/Xenacoelomorph-Neuropeptides-Evolution/blob/main/04-Xenacoelomorph_neuropeptide_GPCRs.txt) contains xenacoelomorph neuropeptide GPCR protein sequences used in this study.

[05-Supplementary-Tables.pdf](https://github.com/faguil/Xenacoelomorph-Neuropeptides-Evolution/blob/main/05-Supplementary-Tables.pdf) contains supplementary tables used in this study.

- **Supplementary Table 1.** Distribution of plesiomorphic bilaterian neuropeptides and neuropeptide receptors in xenacoelomorphs.
- **Supplementary Table 2.** Distribution of novel neuropeptides in xenacoelomorphs and number of detected precursors.
- **Supplementary Table 3.** Affinity of xenacoelomorph GPCRs to protostome and deuterostome sequences of the corresponding receptor types in the RAxML and FastTree sequence analysis.
- **Supplementary Table 4.** Species and accession numbers of sequencing data and tissues/stages of original RNA source.
- **Supplementary Table 5.** Species and accession number of sequencing data or origin of genomes.

[06-Alignments](https://github.com/faguil/Xenacoelomorph-Neuropeptides-Evolution/tree/main/06-Alignments) This folder contains the alignment files used in constructing the phylogenetic trees for this study.

[07-Phylogenetic-trees](https://github.com/faguil/Xenacoelomorph-Neuropeptides-Evolution/tree/main/07-Phylogenetic-trees06-treeBase-phylogenetic-tree-files) This folder contains all the phylogenetic trees generated in this study using FastTree and RAxML software.

[08-CLANS-clustermaps](https://github.com/faguil/Xenacoelomorph-Neuropeptides-Evolution/tree/main/08-CLANS-clustermaps) This folder contains cluster analysis of rhodopsin- and secretin-type neuropeptife GPCRs. **Some files are quite large, so the download may take some time to start.** Please be patient, these files are securely stored in GitHub Large File Storage (LFS), and the download will begin shortly.

[09-Xenacoelomorph-neuropeptide-sequences](https://github.com/faguil/Xenacoelomorph-Neuropeptides-Evolution/tree/main/09-Xenacoelomorph-neuropeptide-sequences) This folder contains xenacoelomorph neuropeptide precursors and neuropeptide GPCR sequences identified in this study.

[10-Mass-spectrometric-data](https://github.com/faguil/Xenacoelomorph-Neuropeptides-Evolution/tree/main/10-Mass-spectrometric-data) This folder contains bioinformatic processing of mass spectrometric data from peptide extracts of three acoel species.
